README.rst

```diff
@@ -2,108 +2,260 @@
 plone.staticresources
 =====================
 
-Static JS and CSS resources for Plone.
+This add-on contains all static JavaScript and CSS resources used by Plone.
 
 
-This add-on contains all JavaScript and CSS resources used by Plone.
+Target audience
+---------------
 
-Prior to Plone 5.2, they were located in ``static/`` directory of ``Products.CMFPlone`` package.
+This documentation aims at:
 
-Due to the specific tooling and workflows used to build frontend resources, and also the different maintenance and release needs of ``Products.CMFPlone`` itself, Plone static resources are now distributed in this package.
-For a complete list of reasons, read the `PLIP 1653 <https://github.com/plone/Products.CMFPlone/issues/1653>`_.
+- **Theme Authors:** who want to change the look and feel of a Plone site and need to:
 
-.. note::
+  - understand how to correctly bundle changes made into existing or new frontend code.
 
-  A note on yarn: we require the use of yarn, because it supports installing packages in a custom named directory, which npm does not (even using ``--prefix``, ``node_modules`` is hardcoded).
-  Our package directory is located at: ``src/plone/staticresources/static/components``.
+- **Integrators and Developers:** who want to customize or maintain the front end of a Plone site for their customers:
 
+  - upgrade existing versions or install new ``npm`` packages.
 
-Target audience
----------------
+  - integrate new JavaScript/CSS behavior, eventually wrapped in a ``pattern`` for optimal integration and reuse within
+    Plone front end ecosystem.
 
-This documentation aims at:
+- **Plone Core Developers:** who want to fix Plone bugs or enhance this add-on.
 
-Integrators and Developers
-^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Who want to customize or maintain the front end of a Plone site for their customers:
+How to upgrade the resources in this package
+--------------------------------------------
 
-- upgrade existing versions or install new ``npm`` packages.
+1. Increase ``npm`` package versions in ``package.json``, in sections ``dependencies`` or ``devDependencies``.
```

Comment:
npm should not be wrapped with double backticks when referring to its proper name. When used as a command or inline literal, then it should be lowercased and wrapped with double bacticks.